includes/class-assets.php

```diff
@@ -515,7 +511,17 @@ private static function enqueue_floating_links( $plugin_url, $version ) {
 		wp_enqueue_script( 's11-floating-links-config', $plugin_url . '/js/packages/floating-links/config.js', array( 'wp-i18n', 'wpbdp-admin-js' ), $version, true );
 		wp_set_script_translations( 's11-floating-links-config', 's11-' );
 		$floating_links_data = array(
-			'proIsInstalled' => WPBDP_Admin_Education::is_installed( 'premium' ),
+			'navLinks'         => array(
+				'freeVersion' => array(
+					'upgrade'       => wpbdp_admin_upgrade_link( 'floating-links' ),
+					'support'       => 'https://wordpress.org/support/plugin/business-directory-plugin/',
+					'documentation' => wpbdp_admin_upgrade_link( 'floating-links', 'get-help/' ),
+				),
+				'proVersion'  => array(
+					'support_and_docs' => wpbdp_admin_upgrade_link( 'floating-links', 'get-help/' ),
```

Comment:
I've pushed a commit addressing your feedback @stephywells and covering all items, except for a few where I've added comments.
I have another question: When a user activates the premium version, they will be sent a link containing the `utm_campaign=liteplugin` parameter for tracking. Does this look good to you, or should it be changed?

Reply:
That's fine, since it's still in the lite plugin code. Thanks for checking!